lib/commands/timeout.js

```diff
@@ -36,7 +36,10 @@ helpers.startNewCommandTimeout = function (cmd) {
     .then(async () => {
       log.warn(`Shutting down because we waited ` +
                `${this.newCommandTimeoutMs / 1000} seconds for a command`);
-      await this.startUnexpectedShutdown(new Error(`Command ${cmd} timed out!`));
+      let errorMessage = `New Command Timeout of ${this.newCommandTimeoutMs}` +
```

Comment:
if we're going to advertise, we should say that it's seconds, since in this log we show that it is in ms, and that could lead to confusion